R/modelFitting.R

```diff
@@ -457,69 +458,9 @@ compBayes <- function(dat,
                         stringsAsFactors = F)
 
 
-    #redo with population level tables
```

Comment:
It's usually a good idea to delete big commented-out blocks like this, but may want to check with Jonathon if he wants to relocate this code somewhere else

Reply:
I'm OK with deleting it.  If we ever revisit the population level hierarchy it will take more time to figure out what this code says than it would to just remake it.